executor.py

```diff
@@ -61,9 +61,14 @@ def process(self, docs: DocumentArray, parameters: Optional[Dict] = {}, **kwargs
             else:
                 continue
 
-            self._load(doc, uri, samples, as_chunks)
+            try:
+                self._load(doc, uri, samples, as_chunks)
+            except Exception:
+                print(f'load trimesh failed,drop this document:{doc.uri}')
+                doc.blob = None
```

Comment:
Please also work with `as_chunks=True`
```suggestion
            try:
                self._load(doc, uri, samples, as_chunks)
            except Exception as ex:
                self.logger.error(f'load trimesh of doc ({doc.uri}) failed, the exception: {ex}')
                if as_chunks:
                    doc.chunks.clear()
                    
```